.conda/install.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -e
+
+CONDA_INSTALLATION_PATH=$HOME/anaconda
```

Comment:
should we consider changing this to `$CWD` or so? I don't want to pollute/be a bad citizen of the jenkins server and place things outside our workspace